contracts/test/WETH9.sol

```diff
@@ -0,0 +1,756 @@
+// Copyright (C) 2015, 2016, 2017 Dapphub
```

Comment:
Couldn't we use the `canonical-weth` dependency to avoid having this contract here and keep code at a minimum?

Reply:
I guess we can, but we need the contract here and use the same one that is used in periphery and dont think that we want WETH contract to depend from a dependency, we want an immutable version in our source code.